src/platforms/apple/common/configuration/metric-kit.mdx

```diff
@@ -4,9 +4,7 @@ sidebar_order: 20
 description: "This feature, once enabled, subscribes to MXDiagnosticPayload data of MetricKit and it to Sentry."
 ---
 
-<Include name="feature-stage-alpha.mdx" />
-
-Once enabled, this feature subscribes to [MetricKit's](https://developer.apple.com/documentation/metrickit) [MXDiagnosticPayload](https://developer.apple.com/documentation/metrickit/mxdiagnosticpayload) data, converts it to events, and sends it to Sentry. This feature is available on Cocoa 8.0.0 and up.
+Once enabled, this feature subscribes to [MetricKit's](https://developer.apple.com/documentation/metrickit) [MXDiagnosticPayload](https://developer.apple.com/documentation/metrickit/mxdiagnosticpayload) data, converts it to events, and sends it to Sentry. This feature is available on Cocoa 8.14.0 and up.
```

Comment:
```suggestion
The MetricKit integration subscribes to [MetricKit's](https://developer.apple.com/documentation/metrickit) [MXDiagnosticPayload](https://developer.apple.com/documentation/metrickit/mxdiagnosticpayload) data, converts it to events, and sends it to Sentry. This feature is available on Cocoa 8.14.0 and up.
```

Reply:
We talk about MetricKit subscribing to MXDiagnosticPayload data in the intro paragraph and then have the list of what the integration subscribes to (MXHangDiagnostic etc.). I'm wondering if it's possible to combine these 2 somehow? It reads odd to have them separated.